cardano-sl.cabal

```diff
@@ -30,6 +30,12 @@ Flag with-explorer
 
   description: Build with explorer
 
+flag bench
```

Comment:
So `flag` was actually needed, just not passing `--bench` wasn't sufficient to not build benchmark dependencies?